src/class_singleCleaner.py

```diff
@@ -128,10 +129,10 @@ def run(self):
                              "MainWindow",
                              'Alert: Your disk or data storage volume'
                              ' is full. Bitmessage will now exit.'),
-                            True)
+                         True)
                     ))
                     # FIXME redundant?
-                    if shared.daemon or not state.enableGUI:
+                    if shared.thisapp.daemon or not state.enableGUI:
```

Comment:
are you sure?

Reply:
I am pretty sure and haven't tested. But while working on this file. I checked the directory of shared and there is no daemon in shared. And even no daemon is found in shared.thisapp. So I took the reference from line no. 77
" if shared.thisapp.daemon or not state.enableGUI: "

Reply:
I am pretty sure. But while working on this file. I checked the directory of shared and there is no daemon in shared. So I took the reference from line no. 77
" if shared.thisapp.daemon or not state.enableGUI: "

Reply:
It looks like this has been resolved